lib/fiware-iotagent-lib.js

```diff
@@ -350,7 +350,6 @@ exports.middlewares = middlewares;
 
 exports.dataPlugins = {
     expressionTransformation: require('./plugins/expressionPlugin'),
-    bidirectionalData: require('./plugins/bidirectionalData'),
```

Comment:
File lib/plugins/bidirectionalData.js should be also removed?

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1421/commits/724570d8c002f98722e793899494939b58fe8e25